phoenix-core/src/main/java/org/apache/phoenix/mapreduce/CsvBulkImportUtil.java

```diff
@@ -69,8 +69,11 @@ public static void configurePreUpsertProcessor(Configuration conf,
     }
 
     @VisibleForTesting
-    static void setChar(Configuration conf, String confKey, char charValue) {
-        conf.set(confKey, Bytes.toString(Base64.getEncoder().encode(Character.toString(charValue).getBytes())));
+
+    static void setChar(Configuration conf, String confKey, Character charValue) {
+        if(charValue!=null) {
```

Comment:
Can you explain the reason for the null check?

Reply:
org.apache.phoenix.mapreduce.CsvBulkImportUtil.getCharacter(Configuration, String) checks to see if the property is set, and will default to null, when it isn't.
It is a very circumspect way of passing a null value, but that's how the existing code works.